### PR TITLE
[MM-44761] Replace occurrences of 'Subscribe Now' with 'Upgrade Now'

### PR DESCRIPTION
**Pull request by nickmisasi**
#### Summary
Replaces occurrences of "Subscribe Now" with "Upgrade Now" throughout the webapp.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-44761

#### Related Pull Requests

- Has server changes (please link here)
- Has mobile changes (please link here)

#### Release Note

```release-note
None
```

### CODE REVIEW

**Review by AGMETEOR — APPROVED** [review]

**Review by marianunez — COMMENTED** [review]
Changes so far LGTM. Will you include the rest of the changes for that ticket in this PR or separately?

**Review by marianunez — COMMENTED** [review]
Do we need to also change the other 3 places where `Subscribe Now` was updated to open the purchase modal?

**Review by marianunez — APPROVED** [review]
LGTM! Thanks @nickmisasi 👍

**Review by neallred — APPROVED** [review]
:shipit:

**Review by AGMETEOR — APPROVED** [review]

File: components/admin_console/billing/billing_subscriptions/index.tsx [hunk 1]
```diff
@@ -73,17 +70,14 @@ const BillingSubscriptions: React.FC = () => {
...
+    const openPricingModal = useOpenPricingModal();
+
     // show the upgrade section when is a free tier customer
     const onUpgradeMattermostCloud = () => {
         trackEvent('cloud_admin', 'click_upgrade_mattermost_cloud');
-        dispatch(openModal({
-            modalId: ModalIdentifiers.CLOUD_PURCHASE,
-            dialogType: PurchaseModal,
-        }));
+        openPricingModal();
```
[the remark this answers is not in the document]
Comment (AGMETEOR) [reply components/admin_console/billing/billing_subscriptions/index.tsx:78]: oops. I think this was a mistake. Should have been opening purchase modal

**Pull request merged**